lint/build.gradle

```diff
@@ -1,5 +1,4 @@
 apply plugin: 'java-library'
-apply plugin: 'kotlin'
```

Comment:
Why did we remove `kotlin` plugin from the lint `build.gradle` file?

Reply:
The build system was complaining about not finding this plugin in the `lint` module and removing it doesn't seem to break anything.  I may add it back in future if needed.

Reply:
I'm not sure why we want and removed both but unless it doesn't break anything, it works i guess.